config.yml

```diff
@@ -2,11 +2,13 @@
 relaychain:
   image: parity/polkadot:v0.9.16 # the docker image to use
   chain: rococo-local # the chain to use
-  runtimeGenesisConfig: # additonal genesis override
-    configuration:
-      config:
-        validation_upgrade_cooldown: 10
-        validation_upgrade_delay: 10
+  runtime:
```

Comment:
is this breaking change necessary?

Reply:
I think not.

How about this?
```yml
relaychain:
  # ...
  runtimeGenesisConfig:
    configuration:
       config:
         validation_upgrade_cooldown: 10
         validation_upgrade_delay: 10
  runtimeConfig:
    session_length_in_blocks: 100
  env:
    # ...
```

Reply:
that will work. maybe we can just make it `overrides` or something similar to allow arbitrary chain spec overrides to make this more generic

Reply:
done.